Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -43,6 +43,13 @@ public struct TuistCommand: ParsableCommand {
         let executeCommand: () async throws -> Void
         do {
             let processedArguments = Array(processArguments(arguments)?.dropFirst() ?? [])
+
+            // Help
+            if processedArguments.isEmpty || processedArguments.first == "--help" {
```

Comment:
Should'n it be shown only if the only argument is help?
```suggestion
            if processedArguments.isEmpty || processedArguments == ["--help"] {
```

Reply:
On 2.x, `tuist --help foo` displays the help. Also, "help" and "-help" work as well. So if you want to match current behavior, perhaps 
```suggestion
            if processedArguments.isEmpty || ["help", "-help", "--help"].contains(processedArguments.first) {
```

Reply:
If we find a way to default on the existing behaviour, that is not having to call helpRequest we would get the expected behaviour: https://github.com/tuist/tuist/pull/4074#issuecomment-1023974965 let's give it a try?

Reply:
Side note, given we are working on the help, it would be nice to have a single help, containing both the `tuist` and `tuistenv` commands, because from the user point of view they are basically the same command :D

Reply:
Agree to `tuistenv` and `tuist` help we can do that in a separate PR if we want but totally confusing these two things aren't displayed together.

From code level I get why because the `TuistEnv` command is in `TuistEnvKit` which can't be imported by `TuistKit`. But we can easily create a shared module for these two and place the commands there.

Reply:
I've been thinking about proposing getting rid of `tuistenv` altogether but ill make a proper proposal for that.